### ISSUE TITLE
Insufficient color contrast in ConnectionsRedirectNotice

### ISSUE THREAD

**Issue by mikkancso** [grafana/grafana#65153]
In [this PR](https://github.com/grafana/grafana/pull/64885) I turned on the `dataConnectionsConsole` feature toggle by default, and as a result `ConnectionsRedirectNotice` is now visible on two pages: Admin/Datasources and Admin/Plugins.

That [PR's Pa11y check](https://drone.grafana.net/grafana/grafana/106851/3/22) was failing because there are elements with insufficient color contrast in this component.
As I couldn't really figure out how to fix this, I reached out to multiple parties ([1](https://raintank-corp.slack.com/archives/C025WTVRBSN/p1679314652021399), [2](https://raintank-corp.slack.com/archives/C030MMKLH2M/p1679389778282939)). Eventually, we couldn't figure out how to fix this, so I increased the error threshold for Pa11y on these two pages, and merged my PR.

@grafana/user-essentials if you have time, please take a look on this and fix it.

**Steps to reproduce**:

- Decrease the Pa11y error threshold by 2 for the mentioned pages
- Run Pa11y

**Actual Result**:
- Pa11y failing

**Expected Result**
- Pa11y passing

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
Elements must have sufficient color contrast (https://dequeuniversity.com/rules/axe/4.4/color-contrast?application=axeAPI)

**Comment by grafanabot** [comment]
@mikkancso please add one or more appropriate labels. Here are some tips:

- if you are making an issue, TODO, or reminder for yourself or your team, please add one label that best describes the product or feature area. Please also add the issue to your project board. :rocket:

- if you are making an issue for any other reason (docs typo, you found a bug, etc), please add at least one label that best describes the product or feature that you are discussing (e.g. `area/alerting`, `datasource/loki`, `type/docs`, `type/bug`, etc). [Our issue triage](https://github.com/grafana/grafana/blob/main/ISSUE_TRIAGE.md#3-categorizing-an-issue) doc also provides additional guidance on labeling. :rocket:

 Thank you! :heart:

**Comment by eledobleefe** [comment]
I investigated a bit and I think this issue with the color contrast can be related to these external bugs: https://github.com/pa11y/pa11y/issues/633 & https://github.com/pa11y/pa11y/issues/623

As we can see in the following screenshots, using the `WCAG Constras checker` does not though any error in the same area.

<img width="1106" alt="Captura de pantalla 2023-05-03 a las 17 40 19" src="https://user-images.githubusercontent.com/65417731/235967248-af135d2b-86b1-4cea-865d-403d788b4470.png">

<img width="1141" alt="Captura de pantalla 2023-05-03 a las 17 39 30" src="https://user-images.githubusercontent.com/65417731/235967299-43e6ec76-b76b-4a18-a869-1c4888c1dfe7.png">

Taking into account all this info gathered, I close this issue.

**Comment by mikkancso** [comment]
Nice investigation, thanks! :clap:
